lang/yabasic/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name			yabasic
-version			2.763
+version			2.86.6
 revision                1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Done.

Reply:
You will need to push your changes.

Reply:
Oops, misfire with git CLI.  Done.